pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go

```diff
@@ -869,7 +869,7 @@ func (r *Reconciler) runReconcileSeedFlow(
 	if err != nil {
 		return err
 	}
-	dwdEndpoint, dwdProbe, err := defaultDependencyWatchdogs(seedClient, kubernetesVersion, r.ImageVector, seed.GetInfo().Spec.Settings, r.GardenNamespace)
+	dwdWeeder, dwdProber, err := defaultDependencyWatchdogs(seedClient, kubernetesVersion, r.ImageVector, seed.GetInfo().Spec.Settings)
```

Comment:
We should pass `r.GardenNamespace` and use instead of `v1beta1constants.GardenNamespace`, We use it in the tests, https://github.com/gardener/gardener/blob/3840acaaf57955fd65330c83b2b2d5bdaad56179/test/integration/gardenlet/seed/seed/seed_suite_test.go#L212.
Maybe that's a reason the tests fail too.

Reply:
oh okay, will try that , thanks Shafeeque.